eslint-config-opent2t/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "eslint-config-opent2t",
```

Comment:
The eslint documentation for shareable configs  states "Make sure the module name begins with eslint-config-."  This seems to be convention but also allows consumers of the config module to extend it using the shortcut name "opent2t" instead of "eslint-config-opent2t".
I think it makes sense to follow their convention.  What do you think?

---
In reply to: [75412630](https://github.com/openT2T/opent2t-cli/pull/5#discussion_r75412630) [](ancestors = 75412630)